index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>🍔 도 와 줘 햄 식 ! 🍟</h1>
+    </header>
+    <div class="container">
+      <aside class="sideBar">
+        <div class="other-menu">
+          <h2 class="other-menu-title">다른거 먹자🍔</h2>
+          <div class="other-menu-macdonald">맥도날드</div>
+          <div class="other-menu-burgerking">버거킹</div>
```

Comment:
```suggestion
          <button class="other-menu-macdonald">맥도날드</button>
          <button class="other-menu-burgerking">버거킹</button
```
시멘틱 테그를 사용할 수 있는 상황이라면 div, span테그를 최대한 지양하는 것이 좋아요! 여기는 버튼의 특성을 갖고 있는 애들이니까 button테그를 사용하거나, list테그를 사용하면 좋을 것 같아요!

시맨틱 테그를 사용하기 위해서 테그들에 먹혀있는 기본 스타일을 없애는 게 좋은데, 그래서 reset.css를 먹이는 편이에요! 어떤 식으로 먹이는지 위에 링크해둘게요 !